lbz/resource.py

```diff
@@ -55,6 +57,8 @@ def __init__(self, event: dict):
             is_base64_encoded=event.get("isBase64Encoded", False),
             query_params=event["multiValueQueryStringParameters"],
         )
+        self._authz_collector.set_resource(self.get_name())
+        self._authz_collector.set_guest_permissions(self.get_guest_authorization() or {})
```

Comment:
I think `get_guest_authorization()` should return an empty dict by default instead of putting here `or {}` 😉